README.md

```diff
@@ -1 +1,66 @@
-# PersonalProject
\ No newline at end of file
+# The High Score Tracker
+
+## Description
+The High Score tracker will allow a program to digitally keep track of high scores without the need for memorization or pen & paper!
+
+## Overview
+The High Score Tracker keeps a record of the high scores for a game. After a game ends, the player will enter their name and score using their keyboard. The player’s score is added to a high score list in the correct location. Lastly, the updated high score list is output to the console. 
+
+## High-Level Design
+### Flowchart
+![A Flowchart](Flowchart.png)
```

Comment:
This should be `flowcharts/Flowchart.png` since your image is in the `flowcharts` directory.